cmd/crowdsec-cli/decisions.go

```diff
@@ -269,22 +267,17 @@ Args :
 			log.Info("Decision successfully added")
 		},
 	}
+	cmdDecisionsAdd.Flags().StringVarP(&Duration, "duration", "d", "4h", "Decision duration (ie. 1h,4h,30m)")
+	cmdDecisionsAdd.Flags().StringVarP(&Scope, "scope", "s", types.Ip, "Decision scope (ie. ip,range,username)")
+	cmdDecisionsAdd.Flags().StringVarP(&Reason, "reason", "R", "", "Decision reason (ie. spam)")
+	cmdDecisionsAdd.Flags().StringVarP(&Type, "type", "t", "ban", "Decision type (ie. ban,captcha,throttle)")
```

Comment:
isn't this flag redundant with the main flag of `cmdDecisions` ?

Reply:
yes that,s the plan, to override the defaults